presto-main/src/main/java/com/facebook/presto/SystemSessionProperties.java

```diff
@@ -1082,6 +1083,11 @@ public SystemSessionProperties(
                         "Enable pushing of filters and projections inside common table expressions.",
                         featuresConfig.getCteFilterAndProjectionPushdownEnabled(),
                         false),
+                integerProperty(
+                        CTE_HEURISTIC_REPLICATION_THRESHOLD,
+                        "Used with CTE Materialization Strategy = Heuristic. CTES are only materialized if they are used greater than or equal to this number",
```

Comment:
nit: perhaps change comment to "if they are used more than this number of times"